giftless/transfer/basic_streaming.py

```diff
@@ -87,6 +87,8 @@ def get(self, organization, repo, oid):
 
         if self.storage.exists(path, oid):
             file = self.storage.get(path, oid)
+            mime_type = self.storage.get_mime_type(path, oid)
```

Comment:
Probably this should be based on `filename` and not `path` (see my comment above)

Reply:
I think this is a different case because the rest of the storage methods receive a path (a `prefix`), and that's what you probably will need when creating the client to get the mime type:

https://github.com/datopian/giftless/pull/90/files#diff-35b02789923e7e0f09735fcb0d51a74799e85ad467ee708372f68b9995816a4fR68

If not backends can parse the path if needed

Reply:
Oh, right, missed that.